README.md

```diff
@@ -121,24 +121,35 @@ A CSV file will also be created along with the html dashboard which will contain
 - Lineage
 - Estimated Advantage
 - Bootstrap 95% interval
 ---
 ### Read analysis tools
 We now provide tools for the analysis of indexed bam files given a set of mutations of interest.
 ```
-freyja extract [query-mutations.csv] [input-bam] --output [directory-of-output-file] --same_read
+freyja extract [query-mutations.csv] [input-bam] --output [outputname.bam] --same_read
 ```
 The above command will extract reads containing one or more mutations of interest and save them to `[input-bam]_extracted.bam`. Additionally, the `--same_read` flag can be included to specify that all query mutations must occur on the same read. In order to exclude reads containing one or more mutations, use the following:
 ```
-freyja filter [query-mutations.csv] [input-bam] [min-site] [max-site] --output [directory-of-output-file]
+freyja filter [query-mutations.csv] [input-bam] [min-site] [max-site] --output [outputname.bam]
 ```
-Where ```[min-site]``` and ```[max-site]``` specify the range of reads to include. Formatting requirements for `[query-mutations.csv]` can be found below.
+Where ```[min-site]``` and ```[max-site]``` specify the range of reads to include. Formatting requirements for `[query-mutations.csv]` can be found below, where SNPs, insertions, and deletions are listed in separate lines.
 ```
 C75T,G230A,A543C
 (732:'TT'),(1349:'A'),(12333:'A')
 (1443:32),(1599:2),(2036:3)
 ```
-Where SNPs, insertions, and deletions are listed in separate lines.
+In many cases, it can be useful to study covariant mutations (i.e. mutations co-occurring on the same read). This can be performed with the command:
+```
+freyja covariants [input_bam] [min_site] [max_site] --output [outputname.tsv]
+```
+This outputs to a tsv file that includes the mutations present in each set of covariants, their frequency (with respect to the total number of reads between `min_site` and `max_site`, inclusively), as well as the coverage ranges. Should the user wish to only consider reads that span the entire genomic region defined by (min_site, max_site), they may include the `--spans_region` flag. Inclusion thresholds for read-mapping quality and the number of observed instances of a set of covariants can be set using `--min_quality` and `--min_count` respectively. The output can then be passed into:
+```
+freyja plot-covariants [covariant_file.tsv] --output [plot-filename(.pdf,.png,etc.)]
+```
+This generates an informative heatmap visualization, showing variants as well as coverage across different reads. The threshold, `--min_mutations`, sets a minimum number of mutations per set of covariants for inclusion in the plot. Example output: 
```

Comment:
adding in an `--nt_muts` options to see all of the nt muts (can leave the translation in the label still) here would be helpful